client/X11/xf_rail.c

```diff
@@ -194,8 +194,8 @@ void xf_rail_end_local_move(xfContext* xfc, xfAppWindow* appWindow)
 	 * we can start to receive GDI orders for the new window dimensions before we
 	 * receive the RAIL ORDER for the new window size.  This avoids that race condition.
 	 */
-	appWindow->windowOffsetX = windowMove.left;
-	appWindow->windowOffsetY = windowMove.top;
+	appWindow->windowOffsetX = (INT16)windowMove.left;
+	appWindow->windowOffsetY = (INT16)windowMove.top;
```

Comment:
This is pre-existing, but this is here to proactively update the new location locally that we are sending to the  server immediatey. I think we should also update the value of "appWindow->localWindowOffsetCorrX" and "appWindow->localWindowOffsetCorrY" here as well.